.gitignore

```diff
@@ -19,3 +19,4 @@
 /public/*
 
 .byebug_history
+/config/secrets.yml
```

Comment:
Can you provide `/config/secrets.yml.example` with some "placeholder" values? Right now we don't have starting point for new developers.